modules/installation-aws-permissions.adoc

```diff
@@ -9,287 +9,172 @@ When you attach the `AdministratorAccess` policy to the IAM user that you create
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:GetObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:PutObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:DeleteObject`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:ListBucketMultipartUploads`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-registry`
-|`s3:AbortMultipartUpload`
-|Yes
-|Working with registry bucket
-
-|`openshift-image-ingress`
-|`iam:GetUser`
-|No
-|Fetching self
-
-|`openshift-image-ingress`
-|`elasticloadbalancing:DescribeLoadBalancers`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`route53:ListHostedZones`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`route53:ChangeResourceRecordSets`
-|Yes
-|Ingress creates and removes names associated with ELB
-
-|`openshift-image-ingress`
-|`tag:GetResources`
-|Yes
-|Used to find appropriate load balancer
-
-|===
+.Required EC2 permissions for installation
+* `ec2:AllocateAddress`
+* `ec2:AssociateAddress`
+* `ec2:AssociateDhcpOptions`
+* `ec2:AssociateRouteTable`
+* `ec2:AttachInternetGateway`
+* `ec2:AuthorizeSecurityGroupEgress`
+* `ec2:AuthorizeSecurityGroupIngress`
+* `ec2:CopyImage`
+* `ec2:CreateDhcpOptions`
+* `ec2:CreateInternetGateway`
+* `ec2:CreateNatGateway`
+* `ec2:CreateRoute`
+* `ec2:CreateRouteTable`
+* `ec2:CreateSecurityGroup`
+* `ec2:CreateSubnet`
+* `ec2:CreateTags`
+* `ec2:CreateVpc`
+* `ec2:CreateVpcEndpoint`
+* `ec2:CreateVolume`
+* `ec2:DeleteSnapshot`
+* `ec2:DeregisterImage`
+* `ec2:DescribeAccountAttributes`
+* `ec2:DescribeAddresses`
+* `ec2:DescribeAvailabilityZones`
+* `ec2:DescribeDhcpOptions`
+* `ec2:DescribeImages`
+* `ec2:DescribeInstanceAttribute`
+* `ec2:DescribeInstanceCreditSpecifications`
+* `ec2:DescribeInstances`
+* `ec2:DescribeInternetGateways`
+* `ec2:DescribeKeyPairs`
+* `ec2:DescribeNatGateways`
+* `ec2:DescribeNetworkAcls`
+* `ec2:DescribePrefixLists`
+* `ec2:DescribeRegions`
+* `ec2:DescribeRouteTables`
+* `ec2:DescribeSecurityGroups`
+* `ec2:DescribeSubnets`
+* `ec2:DescribeTags`
+* `ec2:DescribeVpcEndpoints`
+* `ec2:DescribeVpcs`
+* `ec2:DescribeVpcAttribute`
+* `ec2:DescribeVolumes`
+* `ec2:DescribeVpcClassicLink`
+* `ec2:DescribeVpcClassicLinkDnsSupport`
+* `ec2:ModifyInstanceAttribute`
+* `ec2:ModifySubnetAttribute`
+* `ec2:ModifyVpcAttribute`
+* `ec2:RevokeSecurityGroupEgress`
+* `ec2:RunInstances`
+* `ec2:TerminateInstances`
+* `ec2:DeleteDhcpOptions`
+* `ec2:DeleteRoute`
+* `ec2:RevokeSecurityGroupIngress`
+* `ec2:DisassociateRouteTable`
+* `ec2:ReplaceRouteTableAssociation`
+* `ec2:DeleteRouteTable`
+* `ec2:DeleteSubnet`
+* `ec2:DescribeNetworkInterfaces`
+* `ec2:ModifyNetworkInterfaceAttribute`
+* `ec2:DeleteNatGateway`
+* `ec2:DeleteSecurityGroup`
+* `ec2:DetachInternetGateway`
+* `ec2:DeleteInternetGateway`
+* `ec2:ReleaseAddress`
+* `ec2:DeleteVpc`
```

Comment:
These last six (`DeleteNatGateway` and later) are only needed for `destroy cluster`.